vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -614,6 +614,16 @@
       "source": "parts/linux/cloud-init/artifacts/localdns-delegate.conf",
       "destination": "/home/packer/localdns-delegate.conf"
     },
+    {
+      "type": "file",
+      "source": "parts/linux/cloud-init/artifacts/10_azure_nvidia",
+      "destination": "/home/packer/10_azure_nvidia"
```

Comment:
should we name these files 10_azure_nvidia_gb200? Since it's GB 200 specific.

Reply:
it's not. it's azure_nvidia kernel specific, which will be used for GB300 and the like.  it was confirmed by OS team